src/modal/Modal.svelte

```diff
@@ -35,12 +35,14 @@
         case "chrome":
         case "chromium":
         case "electron":
-        case "microsoft edge":
```

Comment:
should we keep this in case "edge" for a wallet is not defined? As you can simply use the chrome store with edge?

Reply:
now falling-back to "chrome" in case browser is "edge" and provider have no "edge" discovery link